addons/website_event_crm/views/event_crm_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="event_crm_rule_view_tree" model="ir.ui.view">
+        <field name="name">event.crm.rule.view.tree.inherit.website.event.crm</field>
+        <field name="model">event.crm.rule</field>
+        <field name="inherit_id" ref="event_crm.event_crm_rule_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='lead_creation_basis']" position="attributes">
+                <attribute name="invisible">0</attribute>
```

Comment:
I don't really understand why those fields are invisible by default..

Reply:
Actually seems complicated, so much overrides for few result. I would suggest to convince LNA to simplify everything and display options directly in base module to avoid unnecessary view override,; allowing to remove website_event_crm completely normally.